abusehelper/bots/abusech/ransomwarebot.py

```diff
@@ -0,0 +1,59 @@
+import idiokit
+from abusehelper.core import bot, utils
+
+
+COLUMNS = ("first seen", "threat", "malware", "host", "url", "status", "registrar", "ip", "asn", "country")
+
+
+def _value_split(values):
+    results = set()
+
+    for value in values:
+        results = results | set([x for x in value.split("|") if x])
+
+    return tuple(results)
+
+
+@idiokit.stream
+def _parse():
+    while True:
+        event = yield idiokit.next()
+
+        for key, value in event.items():
+            if not value.strip():
+                event.pop(key)
```

Comment:
```event.pop()``` returns the corresponding values for the given key. The returned value is not used in this case, so how would it sound using ```event.clear()``` for consistency?